### PR TITLE
add bagging if variable is set

### PR DESCRIPTION
**Pull request by carebare47**
## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropriate.

## Checklist

If the task is applicable to this pull request (see applicability criteria in brackets), make sure it is completed before checking the corresponding box. Otherwise, tick the box right away. Make sure that **ALL** boxes are checked **BEFORE** the PR is merged.

- [ ] Manually tested that added code works as intended (if any functional/runnable code is added).
- [ ] Added automated tests (if a new class is added (Python or C++), interface of that class must be unit tested).
- [ ] Tested on real hardware (if the changed or added code is used with the real hardware).
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc.)

### CODE REVIEW

File: sr_robot_launch/launch/sr_ur_arms_hands.launch [hunk 1]
```diff
@@ -214,5 +214,5 @@
 
   <!-- Publish underactuation error -->
   <node name="error_reporter" pkg="sr_error_reporter" type="error_reporter" output="screen"/>
-
+
```
Comment (giusebar) [inline sr_robot_launch/launch/sr_ur_arms_hands.launch]: can you remove this space?

**Review by giusebar — CHANGES_REQUESTED** [review]

File: sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch [hunk 2]
```diff
@@ -188,4 +188,14 @@
     <node name="arm_group_position_controller_spawner" pkg="controller_manager" type="spawner" output="screen"
           args="ra_trajectory_controller la_trajectory_controller"/>
   </group>
+
+  <arg name="record_hand_data" value="$(optenv SR_RECORD_HAND_DATA disabled)"/>
+  <arg name="log_directory" value="$(optenv HOME)/shadow_bag_storage/sr_record_all_hand_data"/>
```
Comment (biofotis) [inline sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch:193]: do we need to create this folder in advance?

File: sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch [hunk 3]
```diff
@@ -188,4 +188,14 @@
...
+  <arg name="log_split_size" default="64"/>
+  <arg name="log_topics" value='-e ".*(diagnostics|jog_arm|joint|rh|lh|sh|tf).*" /mechanism_statistics /ros_ethercat/motors_halted /sr_watchdog'/>
```
Comment (biofotis) [inline sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch]: are we sure we want to record sr_watchdog on nuc side? I thought we are avoiding that to reduce traffic

**Review by biofotis — CHANGES_REQUESTED** [review]

File: sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch (hunk 2 above)
Comment (carebare47) [reply sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch:193]: Aurora is handling that

File: sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch (hunk 3 above)
Comment (carebare47) [reply sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch]: Oh yes, that's a really good point, we should figure out what we want that string to be

**Review by biofotis — APPROVED** [review]

**Pull request merged**